scene/gui/control.cpp

```diff
@@ -660,6 +660,8 @@ void Control::_clear_size_warning() {
 //moved theme configuration here, so controls can set up even if still not inside active scene
 
 void Control::add_child_notify(Node *p_child) {
+	CanvasItem::add_child_notify(p_child);
```

Comment:
I wonder if this should be called first or last as a general rule (I don't have the answer).

Reply:
The other derived Control nodes already generally do that, only TabContainer puts it lower then the top and that's only for its internal TabBar.